Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM p4lang/third-party:latest
+MAINTAINER Seth Fowler <seth.fowler@barefootnetworks.com>
+
+# Install dependencies.
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && \
+    apt-get install -y \
+      libjudy-dev \
+      libgmp-dev \
+      libpcap-dev \
+      libboost-program-options-dev \
+      libboost-system-dev \
+      libboost-filesystem-dev \
+      libboost-thread-dev
+
+# Copy entire repository.
+COPY . /behavioral-model/
+WORKDIR /behavioral-model/
+
+# Build.
+RUN ./autogen.sh && \
+    ./configure && \
```

Comment:
Please run configure with `--with-pdfixed` or it limits what can be done with this docker image. You should probably also add `-with-stress-tests` if we are later going to run the tests in the docker.